recipe/meta.yaml

```diff
@@ -11,9 +11,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script:
-    - python setup.py install --single-version-externally-managed --record=record.txt
+  number: 1000
+  script: "{{ PYTHON }} -m pip install . -vvv"
```

Comment:
The build seems to download a lot of dependencies through `pip`, better use `python -m pip install --no-deps --ignore-installed .` here